src/Uniphar.Dotnet.Cli/Token/TokenService.cs

```diff
@@ -0,0 +1,33 @@
+﻿
+namespace Uniphar.Dotnet.Cli.Token;
+
+internal interface ITokenService
+{
+    Task<string> GetAccessTokenAsync(string tenantId, string clientId);
+}
+
+internal class TokenService : ITokenService
+{
+    private const string Scope = ".default";
```

Comment:
The reason why I didn't add scope as an option it was because the cases that we will mainly use the tool for are APIs without scopes. But I could add it as an option maybe.